### DISCUSSION TITLE
STLファイルのモデルを取り込みで，Dualの水粒子がモデルを貫通してしまう。（対策求む）

### DISCUSSION THREAD

**Discussion by Akihiro3818** [DualSPHysics/DualSPHysics#256] [DUALSPHYSICS_V5.2]
Dualの粒子法を用いて水槽モデルを作成し，水槽実験の解析を行っています。外部からモデルを取り込むのに，VTKファイルにより取り込みました。しかし，水粒子がSTLモデルを貫通してしまいます。貫通しないようにするにはどうすればよいでしょうか。

**Comment by alexbexe** [comment]
sure
